README.md

```diff
@@ -158,6 +158,7 @@ Official integrations are maintained by companies building production ready MCP
 - <img height="12" width="12" src="https://developer.paddle.com/favicon.svg" alt="Paddle Logo" /> **[Paddle](https://github.com/PaddleHQ/paddle-mcp-server)** - Interact with the Paddle API. Manage product catalog, billing and subscriptions, and reports.
 - <img height="12" width="12" src="https://secure.pagos.ai/favicon.svg" alt="Pagos Logo" /> **[Pagos](https://github.com/pagos-ai/pagos-mcp)** - Interact with the Pagos API. Query Credit Card BIN Data with more to come.
 - <img height="12" width="12" src="https://www.paypalobjects.com/webstatic/icon/favicon.ico" alt="PayPal Logo" /> **[PayPal](https://mcp.paypal.com)** - PayPal's official MCP server.
+- <img height="12" width="12" src="https://ww2-secure.pearl.com/static/pearl/pearl-logo.svg" alt="Pearl Logo" /> **[Pearl](https://mcp.pearl.com)** - Official MCP Server to interact with Pearl API. Connect your AI Agents with 12,000+ certified experts instantly.
```

Comment:
I'm getting Unauthorized error at this address. Should point to a code repo.
<img width="320" alt="Screenshot 2025-06-20 at 2 18 37 PM" src="https://github.com/user-attachments/assets/e2c71e9c-3ea2-4ca2-9e2b-fab19c0cc02a" />